airsonic-main/src/main/java/org/airsonic/player/controller/MainController.java

```diff
@@ -47,13 +48,22 @@
     private SecurityService securityService;
     @Autowired
     private SettingsService settingsService;
+    @Autowired
+    private MediaFileService mediaFileService;
+    @Autowired
+    private PodcastService podcastService;
 
     @GetMapping
     protected ModelAndView handleRequestInternal(HttpServletRequest request, HttpServletResponse response) throws Exception {
         Map<String, Object> map = new HashMap<>();
 
         String username = securityService.getCurrentUsername(request);
         UserSettings userSettings = settingsService.getUserSettings(username);
+        int[] requestParameterIds = ServletRequestUtils.getIntParameters(request, "id");
+
+        if (requestParameterIds != null && requestParameterIds.length == 1 && isPodcast(requestParameterIds[0])) {
```

Comment:
I'm not quiet sure, why we could have multiple ids set in the url. I tested it out, it's working, but I've never seen this in "wildlife". To make this fix not overcomplicated (ids could be mixed with albums and podcast channels), I'm only handling it when there's only one id.